### PR TITLE
Make the Readme File More Valuable

### LINKED ISSUE #3: Our Second Issue

### PR DESCRIPTION
**Pull request by TimCoreyDemo**
Created a much better Readme layout as well as an example readme layout so that we can better inform people
about what our repository is all about.

Fixes #3

### CODE REVIEW

File: README.md
Comment (TimCorey) [inline README.md:1]: Looks good to me

File: ReadmeStructure.md
Comment (TimCorey) [inline ReadmeStructure.md:1]: I don't like the color green.

**Review by TimCorey — CHANGES_REQUESTED** [review]
Address what I said.

File: ReadmeStructure.md
Comment (TimCoreyDemo) [reply ReadmeStructure.md:1]: Too bad. That is how new lines are indicated.

**Review by TimCoreyDemo — COMMENTED** [review]
I think I am done.

**Review by TimCorey — APPROVED** [review]
OK, fine. It looks good now.

**Comment by TimCoreyDemo** [comment]
Fixes #3

**Pull request merged**